lib/v2/datasource/remote/api/rates_repository.dart

```diff
@@ -16,4 +16,32 @@ class RatesRepository extends NetworkRepository {
             }))
         .catchError((error) => mapHttpError(error));
   }
+
+  Future<Result> getFiatRatesAlternate() async {
+    print("[http] get alternate fiat rates");
+
+    return http
+        .get("http://data.fixer.io/api/latest?access_key=${Config.fixerApiKey}&symbols=CRC,GTQ,USD")
+        .then((http.Response response) => mapHttpResponse(response, (dynamic body) {
+              return FiatRateModel.fromJsonFixer(body);
+            }))
+        .catchError((error) => mapHttpError(error));
+  }
+
+  Future<Result> getUSDRate() async {
+    print('[http] get seeds rate USD');
+
+    final rateURL = '$baseURL/v1/chain/get_table_rows';
+
+    var request = '{"json":true,"code":"tlosto.seeds","scope":"tlosto.seeds","table":"price"}';
+
+    var headers = <String, String>{'Content-type': 'application/json'};
```

Comment:
do we need to declare these? Isnt this part of the base class thats being extended ?

Reply:
It is true forget that the headers are already in the NetworkRepo